pkgs/misc/drivers/epsonscan2/default.nix

```diff
@@ -127,8 +130,8 @@ stdenv.mkDerivation {
     + lib.optionalString withGui ''
       # The icon file extension is .ico but it's actually a png!
       mkdir -p $out/share/icons/hicolor/{48x48,128x128}/apps
-      convert $src/Resources/Icons/escan2_app.ico -resize 48x48 $out/share/icons/hicolor/48x48/apps/epsonscan2.png
-      convert $src/Resources/Icons/escan2_app.ico -resize 128x128 $out/share/icons/hicolor/128x128/apps/epsonscan2.png
+      convert $src/Resources/Icons/escan2_app.ico -resize 48x48 -delete 1,2,3 $out/share/icons/hicolor/48x48/apps/epsonscan2.png
+      convert $src/Resources/Icons/escan2_app.ico -resize 128x128 -delete 1,2,3 $out/share/icons/hicolor/128x128/apps/epsonscan2.png
```

Comment:
icons havent worked, as the ico contains multiple resolutions and resulting in multiple pngs after convertion and thus not being named correctly